MorphicServer/PendingEmail.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Raising the Floor - International
+//
+// Licensed under the New BSD license. You may not use this file except in
+// compliance with this License.
+//
+// You may obtain a copy of the License at
+// https://github.com/GPII/universal/blob/master/LICENSE.txt
+//
+// The R&D leading to these results received funding from the:
+// * Rehabilitation Services Administration, US Dept. of Education under 
+//   grant H421A150006 (APCP)
+// * National Institute on Disability, Independent Living, and 
+//   Rehabilitation Research (NIDILRR)
+// * Administration for Independent Living & Dept. of Education under grants 
+//   H133E080022 (RERC-IT) and H133E130028/90RE5003-01-00 (UIITA-RERC)
+// * European Union's Seventh Framework Programme (FP7/2007-2013) grant 
+//   agreement nos. 289016 (Cloud4all) and 610510 (Prosperity4All)
+// * William and Flora Hewlett Foundation
+// * Ontario Ministry of Research and Innovation
+// * Canadian Foundation for Innovation
+// * Adobe Foundation
+// * Consumer Electronics Association Foundation
+
+using System;
+using System.Threading.Tasks;
+
+namespace MorphicServer
+{
+    /// <summary>
+    /// Class for Pending Emails.
+    ///
+    /// TODO Implement some background task to actually send these.
+    /// </summary>
+    public class PendingEmail : Record
+    {
+        public string UserId { get; set; }
+        public string ToEmail { get; set; }
+        public string FromEmail { get; set; }
+        public string EmailText { get; set; }
```

Comment:
We'll probably need to use multipart HTML+plaintext email.  What do you think if we instead stored some kind of short email identifier like `"AccountVerification"` and a dictionary of parameters like `{"OneTimeToken": someToken}`.  Whatever fetches `PendingEmail`s from the queue can do the work of generating the email content.

Reply:
I'll work on this in a later PR. Totally agree.